cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -179,6 +204,86 @@ describe('Dashboard', () => {
     });
   });
 
+  describe('Draggable widgets', () => {
+    const gutter = 15;
+    const colWidth = 200;
+
+    beforeEach(function () {
+      cy.viewport(gutter + 6 * colWidth, 800);
```

Comment:
Better suited for `before` but it gets reset at the end of each test.
@gabrieldutra is this a bug or a feature?

Reply:
Not sure, but most of cypress state is reset on the end of each test, so my guess is that it's a feature. Also in the examples they never use `before` for this.